index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>PORTFOLIO</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    <header class="header_container">
+        <nav class="navbar">
+            <h2>Welcome</h2>
```

Comment:
Your logo should be placed outside the `nav` tag since it is not a part of the navbar. Also, it would be a good idea to wrap the logo in an `a` tag so that when the visitors click the logo, they are taken back to the top/home screen.

Reply:
@muneebulrehman pls do you mean the h2 tag should be outside my nav?